### ISSUE TITLE
Progress bar for Readability Analyzer not working

### ISSUE THREAD

**Issue by IshiRenga** [kaegi/MorphMan#301]
I noticed this issue with when I had originally upgraded to the Qt 6 version of Anki, which seemed to have broken all of the progress bars for morphman, here being the readability analyzer one:
![image](https://github.com/kaegi/MorphMan/assets/100808313/683c0cba-289a-492a-b7d6-57cad189c980)
In the most recent update, the one for recalcing was fixed, but this one was not.

Version ⁨2.1.61 (0c1eaf4c)⁩
Python 3.9.15 Qt 6.4.2 PyQt 6.4.0

**Comment by mortii** [comment]
The onAnalyze() runs on the main thread which causes the UI to freeze.  I'll try to fix it when I have the time. Does the rest of the Readability Analyzer work fine for you?

**Comment by IshiRenga** [comment]
Yes, the Readability Analyzer works correctly otherwise. Only the progress bar doesn't work for me.
